movie/src/components/Card/index.jsx

```diff
@@ -64,20 +72,20 @@ const Card = ({ movies }) => {
                   />
                 </Item>
                 <Item>
-                  <img src={IMAGE_URL} alt="" />
+                  <img
+                    className="imgOnCard"
+                    src={IMAGE_URL}
+                    alt="ImageOfCard"
```

Comment:
since the `alt` is use to show to the user. it would be better if we write like a regular word `Image of card`

Reply:
Yep, got it!